asciidoc/courses/app-typescript/modules/1-driver/lessons/5-c-writing-data/lesson.adoc

```diff
@@ -49,19 +49,83 @@ Run the file using `ts-node` to view the result:
 ts-node {lab-file}
 ----
 
-You can check the result in the Neo4j Browser by running the following query:
+If successful, you should see output similar to the following (with your name instead of "Your Name"):
```

Comment:
Providing a view of what the user should see is a good addition.

It gives learners:

1. an indication that they have completed a task successfully
2. confidence that they can progress.